docs/src/main/paradox/bootstrap/details.md

```diff
@@ -123,39 +51,19 @@ even if the DNS system is not completely consistent.
 If however we are talking about an inconsistent DNS lookup response during the Initial Bootstrap, the nodes will be delayed
 forming the cluster as they expect the lookups to be consistent, this is checked by the stable-margin configuration option.
 
-For complete safety of the Initial Bootstrap it is recommended to set the `akka.management.cluster.bootstrap.contact-point-discovery.required-contact-point-nr`
+For complete safety of the Initial Bootstrap it is recommended to set the `contact-point-discovery.required-contact-point-nr`
 setting to the exact number of nodes the initial startup of the cluster will be done. For example, if starting a cluster with
 4 nodes initially, and later scaling it out to many more nodes, be sure to set this setting to `4` for additional safety of
 the initial joining, even in face of an flaky discovery mechanism!
 
 @@@ note
-  It *is* crucial for the nodes to have a consistent view of their neighbours for the Initial Bootstrap.
+  It *is* crucial for the nodes to have a consistent view of their neighbours for the Initial Bootstrap. Otherwise
+  different nodes could self join and start multiple clusters.
 @@@
 
-
-## Recommended Configuration
-
-When using the bootstrap module, there are some underlying Akka Cluster settings that should be specified to ensure
-that your deployment is robust.
-
-Since the target environments for this module are dynamic, that is, instances can come and go, failure needs to be
-considered. The following configuration will result in your application being shut down after 40 seconds if it is unable to
-join the discovered seed nodes. In this case, the orchestrator (i.e. Kubernetes or Marathon) will restart your node
-and the operation will (presumably) eventually succeed. You'll want to specify the following in your `application.conf` file:
-
-```hocon
-akka.cluster.shutdown-after-unsuccessful-join-seed-nodes = 40s
-```
-
-Additionally, nodes can crash causing cluster members to become unreachable. This is a tricky problem as it is not
-possible to distinguish between a network partition and a node failure. To rectify this in an automated manner,
-Lightbend provides [Split Brain Resolver](https://developer.lightbend.com/docs/akka-commercial-addons/current/split-brain-resolver.html)
-as a feature of the Lightbend Subscription. This module has a number of strategies that can ensure that the cluster
-continues to function during network partitions and node failures.
-
 ## Customising Join Behavior
```

Comment:
I think we try to use American spelling: Customizing

Reply:
Okay :)